roles/ffh.mesh_wireguard/templates/addtobatX.j2

```diff
@@ -0,0 +1,11 @@
+[Unit]
+After=sys-subsystem-net-devices-{{ vxlan_iface }}\x2d%i.device
+
+[Service]
+Type=oneshot
+ExecStart=/bin/sh /usr/bin/wait_for_iface.sh {{ vxlan_iface }}-%i unknownisfine
```

Comment:
Better unknown_is_fine instead of unknownisfine?

Reply:
done in a9e26cc